aiogram/dispatcher/middlewares.py

```diff
@@ -35,7 +35,7 @@ def setup(self, middleware):
 
         self.applications.append(middleware)
         middleware.setup(self)
-        log.debug(f"Loaded middleware '{middleware.__class__.__name__}'")
+        log.debug(f"Loaded middleware "+repr(middleware.__class__.__name__))
```

Comment:
For logging is recommended to use %-style formating because when log level is disabled formatter will never be called.
```suggestion
        log.debug("Loaded middleware %r", middleware.__class__.__name__)
```